src/plugins/vis_type_vislib/public/vislib/visualizations/point_series/heatmap_chart.js

```diff
@@ -29,8 +31,10 @@ const defaults = {
  * @param chartData {Object} Elasticsearch query results for this specific chart
  */
 export class HeatmapChart extends PointSeries {
-  constructor(handler, chartEl, chartData, seriesConfigArgs, core) {
-    super(handler, chartEl, chartData, seriesConfigArgs, core);
+  constructor(handler, chartEl, chartData, seriesConfigArgs, uiSettings) {
```

Comment:
nice catch. I see it's really `uiSettings` not a `core`
![image](https://user-images.githubusercontent.com/20072247/108114692-2f955880-70aa-11eb-9dca-b94b11044060.png)

But if you found that wrong type, why you haven't fixed that for all seriesTypes + didn't fix constructor of `PointSeries`?

Reply:
Fixed all seriesTypes. Constructor of PointSeries didn't use it now.